packages/types/src/session.ts

```diff
@@ -0,0 +1,37 @@
+import { IInterview, ILesson, IUser } from "@/index";
+
+export type Row = ILesson.Row | IInterview.Row;
+
+export type Self = ILesson.Self | IInterview.Self;
```

Comment:
This is not true. No need to define these two types. The session is not a table in the database. I understand the motives behind this but is un-needed abstraction.